packages/forms-web-app/src/controllers/full-appeal/type-of-planning-application.js

```diff
@@ -9,8 +9,8 @@ const {
 } = require('@pins/business-rules');
```

Comment:
Nothing to-do with this file and suggestion. 
Will it not be better if the items are are not hardcoded in the view but is sent from Controller like the LPD page? So that we can avoid map-planning-application.js?

Reply:
That's what I thought but the values in the business rules don't correspond directly with what's in the page.

Reply:
We've gone for an interim solution for now with a full fix coming in a subsequent sprint.